src/language-js/utils.js

```diff
@@ -657,7 +657,8 @@ function hasJsxIgnoreComment(path) {
     prevSibling.expression.comments &&
     prevSibling.expression.comments.some(
       (comment) => comment.value.trim() === "prettier-ignore"
-    )
+    ) &&
+    "next"
```

Comment:
??

Reply:
This is to return `"next"` instead of `true`.

Reply:
Sorry, I thought it's a `if`, but better rewrite to `if/else`

```js
return (foo && "next")
```

seems wired to me.

Reply:
Sure, should I do `return (foo ? "next" : false)` instead, if that is more readable?

Reply:
```js
  if (prevSibling &&
    prevSibling.type === "JSXExpressionContainer" &&
    prevSibling.expression.type === "JSXEmptyExpression" &&
    prevSibling.expression.comments &&
    prevSibling.expression.comments.some(
      (comment) => comment.value.trim() === "prettier-ignore"
    )) {
    return "next";
  }

  return false;
```

Reply:
Done.